lib/prawn/font/ttc.rb

```diff
@@ -0,0 +1,35 @@
+# font.rb : The Prawn font class
+#
+# Copyright November 2008, Jamis Buck. All Rights Reserved.
+#
+# This is free software. Please see the LICENSE and COPYING files for details.
+#
```

Comment:
Prawn uses git to track code changes and thus authorship information. Given the nature of a highly collaborative project such a Prawn, this information can get outdated rather quickly. Because of this I'd suggest leaving this comment out.

We acknowledge your contribution and you definitely keep copyright for the code.

If you feel strongly about inclusion of this comment please update it to be factually correct as of now.